devlogs/raisa/daily-logs/2025-03-16.md

```
@@ -0,0 +1,320 @@
+    bulb.turnOff();
+  }
+}
+
+void main() {
+  IncandescantBulb bulb = IncandescantBulb();
+  Room room = Room(bulb);
+  room.switchLightOn();
+}
+```
+
+The code works fine. But let’s say you want to change the incandescant bulb in the room into an LED one. If you do so, you have to modify the room class too which is undesirable. 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)